src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Buttons
+description: >
+  Know why they're pushing your buttons.
+---
+
+## Overview
+
+Buttons are common and vital elements because they help make content interactive across the
+internet. They allow a user to take actions with a single click and are placed throughout a
+page or workflow. Buttons use visual signifiers such as shape, color, and size to show that
+a user can interact with them.
+
+## Interface
+
+**Primary buttons** are for the most important action in a given view. There is typically
+one call-to-action on a page or view. **Default** buttons are for secondary actions.
+**Basic buttons** are for repetitive or deemphasized actions in a given view.
+
+![Tone matrix map](abstract:patterns-buttons-type-usage.png)
+
+## Size
+
+The default button size is medium. In screens that have more information density, use small
+buttons. Large buttons are mostly used in onboarding. Buttons and text inputs within the same
+flow should be the same size.
+
+<BestPractice>
+  <Do imageSource={props.data.buttonsSizeDo.childFile.childImageSharp.fluid} />
+  <Dont imageSource={props.data.buttonsSizeDont.childFile.childImageSharp.fluid} />
+</BestPractice>
```

Comment:
This component was never designed to be shown without inline description text. Also, it seems important to give the user a helpful hint similar to https://garden.zendesk.com/components/avatar#select-the-right-shape-for-the-job